components/side-bar/component-tree/component-view/component-view.tsx

```diff
@@ -34,7 +34,7 @@ export function ComponentView(props: ComponentViewProps<PayloadType>) {
 
   return (
     <NavLink
-      href={`/${node.id}`}
+      href={`/${component.id.fullName}`}
```

Comment:
it's the same thing, no?

Reply:
no. the node id now gets the scope in it as well

Reply:
I see